modules/mapbox/src/mapbox-overlay.ts

```diff
@@ -98,6 +98,7 @@ export default class MapboxOverlay implements IControl {
       })
     });
 
+    map.on('move', this._handleMapMove);
```

Comment:
Doesn't this register a second move handler, on top of the one existing in `getDeckInstance`?

Reply:
Correct